### ISSUE TITLE
Proxy.create is not a function

### ISSUE THREAD

**Issue by dhritzkiv** [mafintosh/mongojs#300]
node.js v6 was released today which enabled `Proxy` support without the `--harmony_proxies` flag.

However, the `Proxy` API has changed from [`Proxy.create(handler)`](https://developer.mozilla.org/en-US/docs/Archive/Web/Old_Proxy_API) to [`new Proxy(target, handler)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy).

I fixed the error locally by editing index.js L7 from 

```js
var p = Proxy.create({
      get: function (obj, prop) {
//...
```

to 

```js
var p = new Proxy({}, {
   get: function (obj, prop) {
//...
```

However, I'm not too familiar with the use of `Proxy`, so I'll let the project maintainers make the PR (especially since there's reference to a "Work around" in the code).

**Comment by hustcer** [comment]
Waiting for a fix.

**Comment by JonnyBurger** [comment]
Made a PR: #301 

Need a quick fix? `npm install @jonny/mongojs` and then `require('@jonny/mongojs')` if you don't want to wait for the PR to be merged.

**Disclaimer:** I don't even know what Proxies are, I just wanted to quickly patch it. I'm not gonna update `@jonny/mongojs` in the future, it will be stuck on 2.3.0.

**Comment by saintedlama** [comment]
Closed by #301

**Comment by BernhardSchlegel** [comment]
ping
